applications/MeshMovingApplication/custom_utilities/fixed_mesh_ale_utilities.cpp

```diff
@@ -91,7 +91,7 @@ namespace Kratos
         mpOriginModelPart = &rOriginModelPart;
 
         // Set the origin process info to the virtual model part
-        mrVirtualModelPart.SetProcessInfo(rOriginModelPart.GetProcessInfo());
```

Comment:
this is also a behaviour change. @rubenzorrilla can we do this?

Reply:
Four of the FSIApplication test failed due to this behaviour change. All four of them are part of the embedded FSI test. The step size of the structural and the fluid solver are different.

Reply:
> Four of the FSIApplication test failed due to this behaviour change. All four of them are part of the embedded FSI test. The step size of the structural and the fluid solver are different.

This problem is addressed [87f0b47](https://github.com/KratosMultiphysics/Kratos/pull/13376/commits/87f0b47095d7e16011aadea62c7b7e22012e2d63)

Reply:
But aren't we speaking about the `ProcessInfo` container? I don't see the relation with the `Parameters`.

Reply:
omg sorry, my mind did a bit of a journey. Just replace every `Parameters` I mentioned with `ProcessInfo`.

Reply:
OK, now everything is clear :smile:. As I said, you're right, we are currently doing a copy. As this is by accident, would you mind to put a //FIXME: comment saying something like "this is now creating a copy and it should be a reference (otherwise the variable time step won't work)". Thanks!